docs/DeveloperGuide.md

```diff
@@ -184,6 +184,23 @@ A `TableNumber` object stores a table number as an integer. It is wrapped in an
   * Pros: Allows for greater flexibility in implementing features that are specific to either guests or vendors.
   * Cons: More code duplication.
 
+### Delete feature
+
+The delete feature allows users to delete a guest or vendor in WedLog, through the respective classes `GuestDeleteCommand` and `VendorDeleteCommand`. The feature makes use of the current `Index` of the person in the displayed list to identify the person.
+
+Given below is an example usage scenario and how the delete mechanism behaves at each step.
+
+Step 1. The user launches the application for the first time. All guests and vendors added during the last use of the app are shown in their respective lists.
```

Comment:
If they are using the application for the first time, they will not have any "last use of the app".
Perhaps scenario can be modified to be about user launching the application (not first time)